packages/iTwinUI-react/package.json

```diff
@@ -52,56 +52,56 @@
     "@types/react-table": "^7.0.18",
     "classnames": "^2.2.6",
     "react-table": "^7.1.0",
-    "react-transition-group": "^4.1.1"
+    "react-transition-group": "^4.4.2"
   },
   "devDependencies": {
     "@babel/core": "^7.12.10",
-    "@storybook/addon-a11y": "^6.4.19",
-    "@storybook/addon-essentials": "~6.4.19",
-    "@storybook/addon-storysource": "~6.4.19",
-    "@storybook/addons": "~6.4.19",
-    "@storybook/api": "~6.4.19",
-    "@storybook/builder-webpack5": "~6.4.19",
-    "@storybook/components": "~6.4.19",
-    "@storybook/core-events": "~6.4.19",
-    "@storybook/manager-webpack5": "~6.4.19",
+    "@storybook/addon-a11y": "^6.5.0-beta.1",
+    "@storybook/addon-essentials": "~6.5.0-beta.1",
+    "@storybook/addon-storysource": "~6.5.0-beta.1",
+    "@storybook/addons": "~6.5.0-beta.1",
+    "@storybook/api": "~6.5.0-beta.1",
+    "@storybook/builder-webpack5": "~6.5.0-beta.1",
+    "@storybook/components": "~6.5.0-beta.1",
+    "@storybook/core-events": "~6.5.0-beta.1",
+    "@storybook/manager-webpack5": "~6.5.0-beta.1",
     "@storybook/preset-typescript": "^3.0.0",
-    "@storybook/react": "~6.4.19",
-    "@storybook/theming": "~6.4.19",
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^12.0.0",
-    "@testing-library/react-hooks": "^7.0.1",
-    "@testing-library/user-event": "^13.5.0",
+    "@storybook/react": "~6.5.0-beta.1",
+    "@storybook/theming": "~6.5.0-beta.1",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.2.0",
+    "@testing-library/user-event": "^14.1.1",
     "@types/classnames": "^2.2.7",
     "@types/jest": "^27.0.0",
     "@types/node": "^10.12.2",
-    "@types/react": "^17.0.3",
-    "@types/react-dom": "^17.0.3",
-    "@types/react-transition-group": "^2.9.2",
+    "@types/react": "^18.0.6",
+    "@types/react-dom": "^18.0.2",
+    "@types/react-transition-group": "^4.4.4",
     "@typescript-eslint/eslint-plugin": "^5.17.0",
     "@typescript-eslint/parser": "^5.17.0",
     "babel-loader": "^8.2.2",
     "concurrently": "^5.3.0",
     "configs": "*",
     "creevey": "^0.8.0-beta.0",
     "inquirer": "^6.2.2",
-    "jest": "^27.0.6",
-    "jest-cli": "^27.0.6",
-    "jest-junit": "^12.0.0",
+    "jest": "^28.1.0",
+    "jest-cli": "^28.1.0",
+    "jest-environment-jsdom": "^28.1.0",
+    "jest-junit": "^13.2.0",
     "markdown-to-jsx": "6.11.4",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
     "storybook-dark-mode": "^1.0.9",
     "tippy.js": "^6.3.1",
-    "ts-jest": "^27.0.4",
+    "ts-jest": "^28.0.2",
     "ts-loader": "^9.2.8",
     "ts-node": "^8.0.2",
     "typescript": "^4.4.3",
     "webpack": "5"
   },
   "peerDependencies": {
-    "react": "^16.8.6 || ^17.0.0",
-    "react-dom": "^16.8.6 || ^17.0.0"
+    "react": ">=16.8.6",
+    "react-dom": ">=16.8.6"
```

Comment:
hmm..changed to a range of versions

Reply:
I just dont like this
![image](https://user-images.githubusercontent.com/81580355/169505866-5756ed2e-bca6-49b4-91d8-35e898f8daa5.png)

Tried with `||` but got same.. So I guess syntax is fine, just something is not liked there :/

Reply:
where is this coming from? I wonder if it's because of deps vs dev deps.

Reply:
I was just runing yarn after all the changes.. At first I tohught its because of wrong syntax, I had..But it had same with the `||`, so I am not sure..
When message does not show full path, kinda hard to guess..